### ISSUE TITLE
Probable bug: evaluation of trained model on MIMIC shows terrible performance

### ISSUE THREAD

**Issue by thibaultmax** [grouchysmurf/medorder_prediction#1]
Running `evaluate_mimic.py` shows terrible prediction accuracy and high loss on the test set. This is surprising considering that cross-validation and training with validation show comparable performance between the training and validation splits, and that the test set is constructed using scikit-learn's train_test_split, which is also used for the validation set when training with validation.

This is probably a bug which should be investigated.

**Comment by thibaultmax** [comment]
Found the bug. The `get_profiles` function of the preprocesor inverted profiles and targets. Will fix this in the medprofile_analysis repo which includes all of the code of this repo.

**Comment by thibaultmax** [comment]
Bug is fixed in medprofile_analysis.
